### PR TITLE
Fix Slack channel information

### PR DESCRIPTION
**Pull request by ebyhr**

### CODE REVIEW

File: slack.md [hunk 1]
```diff
@@ -28,10 +28,10 @@ available for help.
 
 ## Channels
 
-* [#general](https://{{site.slack_fqdn}}/messages/CFLB9AMBN) - discussions about using and developing Trino
+* [#announcements](https://{{site.slack_fqdn}}/messages/CFLB9AMBN) - announcements about Trino
```
Comment (mosabua) [inline slack.md:31]: I thought #general is going to be used for announcements?

Comment (mosabua) [reply slack.md:31]: seems like we renamed .. so is the default channel that everyone automatically now joins #announcements ?

Comment (martint) [reply slack.md:31]: Yes

**Review by mosabua — APPROVED** [review]

**Comment by mosabua** [comment]
Can we merge this @martint ?

**Pull request merged**